packages/react-testing/src/types.ts

```diff
@@ -118,12 +220,15 @@ export interface Node<Props> {
   trigger<K extends FunctionKeys<Props>>(
     prop: K,
     ...args: DeepPartialArguments<Props[K]>
-  ): ReturnType<
```

Comment:
Another case where we're modifying the trigger path. Rational has been reqested above.